modules/codemod/lib/v7/spec/compoundActionBar.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transform from '../compoundActionBar';
+import {stripIndent} from 'common-tags';
+
+const expectTransform = expectTransformFactory(transform);
+
+describe('ActionBar', () => {
+  it('should restructure ActionBar usages with props', () => {
+    const input = stripIndent`
+      import {ActionBar} from '@workday/canvas-kit-react/action-bar'
+
+      <ActionBar fixed="true" />
+    `;
+
+    const expected = stripIndent`
+      import {ActionBar} from '@workday/canvas-kit-react/action-bar'
+
+      <ActionBar />
+    `;
+
+    expectTransform(input, expected);
```

Comment:
Two questions:

- Will this support default imports? `import ActionBar from '@workday/canvas-kit-react/action-bar`
- Will this support imports from `@workday/canvas-kit-react`?

Reply:
- No. There is no default import from action-bar files.
- Yes. It supports imports from main directory.